core/upgrade/upgrades.php

```diff
@@ -3,8 +3,6 @@
 
 use Elementor\Core\Breakpoints\Manager as Breakpoints_Manager;
 use Elementor\Core\Experiments\Manager as Experiments_Manager;
-use Elementor\Core\Kits\Documents\Tabs\Settings_Layout;
-use Elementor\Core\Responsive\Responsive;
```

Comment:
Why?
I assume that those imports weren't in use so the IDE removed them automatically, but just making sure...

Reply:
I don't think it's the IDE, because its looks more like my behavior deleting unused namespaces.
 Anyways I ensured how it was before me:
 
![image](https://user-images.githubusercontent.com/10234691/119257398-d1f33980-bbcd-11eb-94e3-ca038d224a87.png)

Reply:
It looks grayed out, which means that's not in use, maybe you should remove it.
@matipojo  ?

Reply:
Yes. Please remove the unused `use`.